process/process.go

```diff
@@ -280,6 +418,10 @@ func (p *process) PathAccess(path string, aMode domain.AccessMode) error {
 // getInfo retrieves info about the process
 func (p *process) getInfo() error {
 
+	if p.initialized {
```

Comment:
I think p.getInfo() is really p.init(). 

p.getInfo() sounds like "return me the process info", but in fact in does not. It initialized the process object.

Also: do we need locking in the process object?

Reply:
I like p.init() idea, will change.

No, no locking needed as operations over any given process object happen within a unique execution context.